src/change-steps/processChangeSteps.ts

```diff
@@ -159,11 +159,9 @@ export function processChangeSteps(
         const sourceAttrs = oldUpdate?.oldAttrs || c.node.attrs
         const { dataTracked, ...restAttrs } = sourceAttrs
         const oldAttrs = lastChangeRejected ? oldUpdate.oldAttrs : restAttrs
-        // if the node is list, we need to track only the last change. TODO refactor the update-node-attrs to handle all edge cases such as lists.
-        const newDataTracked =
-          c.node.type != c.node.type.schema.nodes.list
-            ? [...oldDataTracked.filter((d) => !oldUpdate || d.id !== oldUpdate.id || lastChangeRejected)]
-            : []
```

Comment:
becuase when reject any change, and then redo the same change it will introduce duplicate changes one rejected and one in suggestion. same as when do a change and undo it by re change the list to the default type it will introduce it as a new change

Reply:
can you give me an example for that case to duplicate issue

Reply:
https://jira.atypon.com/secure/attachment/1783509/1783509_rejecting+the+list+type+change+suggestion.mp4

Reply:
will proceed with that, and work in [TCE-020](https://confluence.atypon.com/display/E2E/Track+changes+extended+improvements) should prevent re-do of rejected/approved change

Reply:
okay, please mention this on the ticket